### PR TITLE
Feature/where in the uk

### PR DESCRIPTION
**Pull request by alexisagos**

### CODE REVIEW

File: src/js/iopps.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ function ioppsToggles() {
...
   });
 
-  $('.sector').on('click', function (event) {
-    var sectorNames = $('.sector-name');
-    for (var i = 0;i < sectorNames.length; i++) {
-      sectorNames[i].innerText = event.target.dataset.sector;
-    }
-    $('#sector-list').hide();
-  });
+  // $('.sector').on('click', function (event) {
```
Comment (mwnicks) [inline src/js/iopps.js]: Why has this been commented out?
Looks like old code we don't need anymore, so should be removed.

File: src/templates/where-to-invest.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            <div class="row">
+              <div class="col-xs-12 col-sm-5">
+                <div class="wti-map">
+                  <img id="clusterMapImage" src="/{{assets}}/media/wti-maps/{{ clusterMap.labelledImage }}">
+                </div>
+              </div>
+              <div class="col-xs-12 col-sm-7">
+                <div class="wti-summary">
+                  <p>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```
Comment (mwnicks) [inline src/templates/where-to-invest.html]: If we are looking to push this live we can't have lorem ipsum in it.

Comment (alexisagos) [inline src/templates/where-to-invest.html]: Done

File: src/js/iopps.js (hunk 1 above)
Comment (alexisagos) [inline src/js/iopps.js]: Yes removed

**Pull request merged**